src/diffusers/schedulers/scheduling_ddpm_flax.py

```diff
@@ -264,7 +268,7 @@ def step(
 
         # 6. Add noise
         def random_variance():
-            split_key = jax.random.split(key, num=1)
+            split_key = jax.random.split(key, ())
```

Comment:
why do we make this change here? `num=1` would still work with `jax.random.key`, no?

Reply:
It will be an error soon in newer versions of JAX, https://github.com/google/jax/blob/343e18fcb693c3f1b2cace56a4faea8fd3e2cadd/jax/_src/random.py#L104-L108.

Reply:
In that case, I'd use:

```suggestion
            split_key = jax.random.split(key, num=(1,))
```

Reply:
That doesn't fix the batching dimension error. In any case, I have done `jax.random.split(key, num=1)[0]` to remove the batch dimension.

Reply:
ah yes you are right!